_includes/nav.html

```diff
@@ -2,16 +2,16 @@
 
 <div class="{% unless homepage %}border-bottom{% endunless %}">
   <div class="container-lg px-3">
-    <div class="py-4 d-sm-flex flex-sm-row">
-      <h1 class="alt-h4 col-sm-4 lh-default v-align-middle">
+    <div class="py-4 d-flex flex-row flex-justify-between">
```

Comment:
`flex-row` is the default, so it isn't needed here.

Reply:
Sorry about that. Fixed